### PR TITLE
PL Features Added

### PR DESCRIPTION
**Pull request by alexawesomecode**
Hello

I have worked on all the requirements. Here is a review of what I did:

1. html forl to look for city weather
2. js call and logic to openweather 
3. ability to display temp, city and description
4. changing colors and images  in relation to temp 
5. toggle button for celsius or farenheit display
6. cool layout!

### CODE REVIEW

File: .eslintrc.json.1 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (MauricioRobayo) [inline .eslintrc.json.1]: Please remove this file from the repository.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "extends": "airbnb-base",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
```
Comment (MauricioRobayo) [inline .eslintrc.json]: You should not modify [this file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+  </h3>
+
+  <p align="center">
+ A Weather App Project for Microverse. Its also part of the <a href="https://www.theodinproject.com/courses/ruby-on-rails/lessons/final-project">Odin project curriculum.</a>
+    <br />
+
+  </p>
+</p>
+
+## Project A Tic Tac Toe Game App
```
Comment (MauricioRobayo) [inline README.md]: Tic Tac Toc 🤔.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "project-weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (MauricioRobayo) [inline package.json]: Include the scripts required by your project, for example, `build`.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "git+https://github.com/alexawesomecode/project-weather-app.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/alexawesomecode/project-weather-app/issues"
+  },
+  "homepage": "https://github.com/alexawesomecode/project-weather-app#readme",
+  "devDependencies": {
```
Comment (MauricioRobayo) [inline package.json:23]: Add all of your project devDependencies here, including Webpack related ones.

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+let country;
+let temp;
+let cityname;
+let tempMetric;
+
+
+const convertTemp = (t) => {
+  const celcius = Math.floor(t - 273.15);
+  const farenheit = Math.floor((celcius * (9 / 5)) + 32);
+  return [celcius, farenheit];
```
Comment (MauricioRobayo) [inline src/index.js]: You should make different requests, please take a look at the `units` option of the Open Weather API: https://openweathermap.org/current#data

```
api.openweathermap.org/data/2.5/find?q=London&units=metric => Celsius 
api.openweathermap.org/data/2.5/find?q=London&units=imperial => Fahrenheit
```

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Well done, nice UI, the background gradient is looking cool!

Please double-check the following before continuing:

* [ ] Update your README.md according to this project, it seems to be from an old Tic Tac Toe project.
* [ ] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).
* [ ] Files not related to the project should not be committed, you have a `.eslintrc.json.1` file ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] All necessarily used packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
* [ ] Necessary run common commands like `build`, `development` should be in `package.json` inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
* [ ] The application makes different requests for the Celsius data and the Fahrenheit data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

Status: __Request changes__.

File: src/index.js (hunk 6 above)
Comment (alexawesomecode) [reply src/index.js]: thanks for this detailed feedback. i will work on the changes. alex

**Review by shloch — APPROVED** [review]
## Every point from former review was corrected, and app still works properly....

### great job

Screen Shot 2020-03-18 at 11 42 00 PM
accomplished

**Pull request merged**